google-cloud-bigtable/lib/google/cloud/bigtable/service.rb

```diff
@@ -734,6 +734,75 @@ def read_modify_write_row table_name, row_key, rules, app_profile_id: nil
           end
         end
 
+        ##
+        # Starts creating a new backup. The underlying Google::Longrunning::Operation tracks creation of the backup.
+        #
+        # @return [Google::Gax::Operation]
+        #
+        def create_backup instance_id, cluster_id, backup_id, source_table_id, expire_time
```

Comment:
More than 2 or 3 positional arguments makes me nervous. Are we confident this set will not change (e.g. we won't have any arguments being added, deprecated, or made optional)? Is the ordering obvious and easily remembered?

Reply:
The first three args are the "path" to the backup. The fourth arg is the source table. And the fifth is a required property of the backup.

Reply:
Since this is an internal method (private class) and the signature does not affect the public API, I'll change this to keyword args without further discussion.